packages/coder/coder/overlay/app-readme.md

```diff
@@ -0,0 +1,16 @@
+# Coder/coder
+## Overview
+Coder provides development environments in your public and private clouds. Environments are defined with Terraform, connected through a secure high-speed tunnel, and are automatically shut down when not in use to save on costs. Coder gives engineering teams the flexibility to use the cloud for workloads that are most beneficial to them.
+
+Developers can connect to Coder workspaces the tools and IDEs they know and love including with VS Code Remote, JetBrains Gateway, SSH as well as web IDEs such as JupyterLab, code-server, RStudio, and web terminals.
```

Comment:
Also, I would remove the "with" in "... and IDEs they know and love including with VS Code Remote..."